scripts/dependency_report.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Show any go dependencies in MicroShift's go.mod that do not appear
+# in other OpenShift components.
+
+set -o errexit
+set -o errtrace
+set -o nounset
+set -o pipefail
+
+REPOROOT="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/..")"
+WORKDIR="${REPOROOT}/_output/dependency_report"
```

Comment:
If we're adding a new directory, we should also clean it in Makefile

Reply:
I don't understand what you mean.

Reply:
`make clean` would not delete this new directory , so _output will remain.